### PR TITLE
add Juan's bio

### PR DESCRIPTION
**Pull request by JuanJovel**

### CODE REVIEW

File: team.html [hunk 1]
```diff
@@ -101,493 +101,509 @@
...
+                                <p class="card-text">
+                                    Juan is a sophomore studying CS at Virginia Tech. He joined WebDVT to expand his
+                                    knowledge
+                                    of web development and programming in general. Juan enjoys listening to music,
+                                    reading, and
+                                    traveling.
+                                </p>
+                                <hr/>
+                                <div class="row footer-social">
+                                    <div class="col-lg-2">
```
Comment (hassanabral) [inline team.html]: Type: it should be `col-lg-12`.

**Review by hassanabral — CHANGES_REQUESTED** [review]
Everything looks good except one minor thing:

In your latest commit **"Fixed a missing closing tag"**, you seem to have changed the styling of the entire `team.html` document. I don't know if you did this intentionally or unintentionally but generally speaking, you should avoid doing this because:

1) Assuming that this is asked for, it should be done in a separate PR.
2) It makes it very hard for the reviewer to pinpoint the "actual" changes you made.

I'd appreciate it if you could revert your latest commit (then make a new commit adding the missing closing tag). Here is the command to use:

```
git reset --soft HEAD~1
```
The `git reset` command will rewind your current HEAD branch to the specified revision. In this example above, we're returning to the one before the current revision - effectively making our last commit undone.

Note the --soft flag: this makes sure that the changes in undone revisions are preserved. After running the command, you'll find the changes as uncommitted local modifications in your working copy.

If you don't want to keep these changes, simply use the --hard flag. Be sure to only do this when you're sure you don't need these changes anymore.
```
git reset --hard HEAD~1
```

**Comment by JuanJovel** [comment]
Sorry, I hit auto format on WebStorm and it did it automatically.

**Comment by hassanabral** [comment]
> Sorry, I hit auto format on WebStorm and it did it automatically.

No worry. I also made this mistake when I did my first internship. A good practice is to check what changes you're committing before running `git commit`. If you're using Webstorm, you can use `ctrl + k` on Windows or `command + k` on Mac and it'll display a GUI window and show you all the file changes that are in the staging area.

**Review by hassanabral — APPROVED** [review]
Thanks for addressing my comments! ✅

**Pull request merged**